src/fabric/src/fabric2_fdb.erl

```diff
@@ -1186,18 +1186,27 @@ seq_to_vs(Seq) when is_binary(Seq) ->
 
 
 next_vs({versionstamp, VS, Batch, TxId}) ->
-    {V, B, T} = case TxId =< 65535 of
+    {V, B, T} = case TxId < 16#FFFF of
```

Comment:
Good use of the hex version there to make it more clear and show that I was off by one.